skatepark/sass/base/_colors.scss

```diff
@@ -0,0 +1,10 @@
+.has-foreground-background-color,
+.has-selection-background-color,
+.has-primary-background-color,
+.has-secondary-background-color {
+	color: var(--wp--preset--color--background);
```

Comment:
Without this (or something like it) patterns and existing content may be less likely to look acceptable when switching themes if they are using semantic colors.  I was thinking less about the application of the colors to the example above and more about how it would apply to an existing pattern.  For instance if the original leveraged the primary color as a background, and the default foreground (text) color was appropriate for that theme's palette.  Sure it's possible to go and change those blocks once the theme has been switched, but it seems like an important goal for that relationship between semantic color combinations be outlined for the themes so that a user wouldn't have to.

However I didn't actually quantify that with an example so it might be a moot point.  I'll see if I can post a good existing example.

Reply:
I see what you're saying @scruffian.  I've backed out my commits about "reasonable defaults".